packages/typespec-go/test/cadlranch/versioning/rettypechangedfromgroup/returntypechangedfrom_client_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package rettypechangedfromgroup_test
+
+import (
+	"context"
+	"rettypechangedfromgroup"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_Test(t *testing.T) {
```

Comment:
Can we make the test name more descriptive of the scenario.

Reply:
The test name comes from the method name on the client. In this case, the method name is `Test`.